test/Microsoft.ML.Tests/Transformers/WordTokenizeTests.cs

```diff
@@ -54,6 +56,17 @@ public void TestCommandLine()
             Assert.Equal(Maml.Main(new[] { @"showschema loader=Text{col=A:TX:0} xf=WordToken{col=B:A} in=f:\2.txt" }), (int)0);
         }
 
+#pragma warning disable 169
```

Comment:
Please document why the warning is safe to disable here.

Reply:
Done.